README.md

```diff
@@ -1384,6 +1386,56 @@ out  | evt.ota_end.report      | object     | Sent on upgrade end with upgrade s
    }
 }
 ```
+### Virtual meter service
+
+This service enables a device to report accumulated consumption in case it does not have its own metering capabilities (eg. a relay with output binary switch service or thermostat). Thanks to the service accumulated consumption is calculated based on manually provided consumption in a unit of time (eg. watts, m3/h). A device shall calculate cumulated consumption at every state's change, mode change and at least every interval value - 30 minutes by default. When this service is added on a device, appropriate metering service is created (eg. meter_elec, meter_water) and a device starts to send measurements.
+
+#### Service names
+
+`meter_virtual`
+
+#### Interfaces
+
+Type | Interface                | Value type |   Unit  | Description
```

Comment:
Once again you're changing the documentation conventions, but introducing different columns to the table. If interfaces use properties then there should be a column called property describing properties used by the interface.